phpBB/includes/functions_download.php

```diff
@@ -248,30 +248,8 @@ function send_file_to_browser($attachment, $category)
 
 		if ($fp !== false)
 		{
-			// Deliver file partially if requested
-			if ($range = phpbb_http_byte_range($size))
```

Comment:
are you removing the partial download on purpose?

Reply:
Yes, that probably don't work with remote files, only with local files, that is why i said that maybe i should use BinaryFileResponse for local storage only, and stream_copy_to_stream for the others

Reply:
ok
actually for remote file it could work but it's more complicated (something like "read fast-forward until you reach the asked bits, then stream them") not sure it is worse it.

Reply:
Yes, that should work, but better for #5274 pr, because is where i moved this code.

Reply:
@Nicofuma If i don't understand wrong, this is a very bad idea, because if a browser request a file by ranges, for each range request you have to read the file.
This can be useful only with local adapter

Reply:
yep, that's why I said I'm not sure it is worse it